hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/protocol/DatanodeProtocol.java

```diff
@@ -81,6 +81,7 @@
   final static int DNA_BALANCERBANDWIDTHUPDATE = 8; // update balancer bandwidth
   final static int DNA_CACHE = 9;      // cache blocks
   final static int DNA_UNCACHE = 10;   // uncache blocks
+  final static int DNA_HASHMISMATCH = 11;   // uncache blocks
```

Comment:
change the comment to correspond to what it is.

Reply:
What?

Reply:
is DNA_HASHMISMATCH = uncache blocks ?